xarray/core/common.py

```diff
@@ -577,13 +577,10 @@ def pipe(
 
         >>> def adder(data, arg):
         ...     return data + arg
-        ...
```

Comment:
@keewis any thoughts on these? I was getting similar corrections from blackdoc. The proposed code may look somwhat bunched?

Reply:
`black` will make sure there's always a newline at the end of the code, so `blackdoc` will always remove that. I could fix that for code blocks, but that would mean I'd have to make it analyze the code. Not sure if that's worth the effort?

Reply:
Why remove the newline at the end of the code? Wouldn't that make for nicer spacing?

(TBC, blackdoc is awesome, and feel free to prioritize these comments as you wish, I'm hoping to be helpful rather than complain 😃)

Reply:
don't worry, that's how I understood it :grin:

I think it might be better to continue this discussion in keewis/blackdoc#52?

Reply:
this should be fixed in the version I released just now

Reply:
I reverted the changes